books/RayTracingTheNextWeek.html

```diff
@@ -2350,9 +2350,9 @@
   $$ \mathbf{n} = \text{unit_vector}(\mathbf{u} \times \mathbf{v}) $$
 
 The plane is defined as all points $(x,y,z)$ that satisfy the equation $Ax + By + Cz = D$. Well, we
-know that $\mathbf{O}$ lies on the plane, so we can use that to find $D$:
+know that $\mathbf{Q}$ lies on the plane, so we can use that to find $D$:
 
-  $$ D = \mathbf{n}_x\mathbf{O}_x + \mathbf{n}_y\mathbf{O}_y + \mathbf{n}_z\mathbf{O}_z $$
+  $$ D = \mathbf{n}_x\mathbf{Q}_x + \mathbf{n}_y\mathbf{Q}_y + \mathbf{n}_z\mathbf{Q}_z $$
```

Comment:
What's our policy on bolding vector components? Do we have one?

Reply:
Oooh, excellent question. Despite the above, I'd say points and vectors are bolded, scalars are not. And "Qz" is a scalar, so should be unbolded. Will fix.

Reply:
Sigh. I suppose I should give a darn and review the entire text...

Reply:
Ok, that wasn't too hard. Turns out that we're consistent everywhere else. Excellent catch.